From 3c5dc577ee6d0a7a8d2d9310295ee760a24df5e5 Mon Sep 17 00:00:00 2001
From: Oliver Neukum <oneukum@suse.com>
Date: Mon, 7 Dec 2020 10:44:10 +0100
Subject: [PATCH] Define VFL_TYPE_VIDEO for g2d
Patch-mainline: Never, compilation fix for 4.12-based kernel
References: git-fixes

The proper dependency for this fix breaks the build in an unfixable
manner, as it renames things. So we revert this locally.

Signed-off-by: Oliver Neukum <oneukum@suse.com>
---
 drivers/media/platform/s5p-g2d/g2d.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/media/platform/s5p-g2d/g2d.c b/drivers/media/platform/s5p-g2d/g2d.c
index a79ec07d3..3cc6c6a45 100644
--- a/drivers/media/platform/s5p-g2d/g2d.c
+++ b/drivers/media/platform/s5p-g2d/g2d.c
@@ -31,6 +31,8 @@
 
 #define fh2ctx(__fh) container_of(__fh, struct g2d_ctx, fh)
 
+#define VFL_TYPE_VIDEO VFL_TYPE_GRABBER
+
 static struct g2d_fmt formats[] = {
 	{
 		.name	= "XRGB_8888",
-- 
2.26.2

